stats/DataAggregation.md

```diff
@@ -0,0 +1,67 @@
+# Data Aggregation API Overview
+The stats library allows users to define measurements how data are aggregated and broken down.
+The core data types used are:
+* `Aggregation`: defines the aggregation type (e.g. `Distribution`, `Sum`, `Mean`).
+* `View`: defines how individual measurements are broken down by tags and aggregated.
+
+## Aggregation
+An Aggregation describes how data collected is aggregated.
+
+The library SHOULD provide support for multiple types of Aggregations:
```

Comment:
This requirement is technically satisfied by an implementation with any two Aggregations--perhaps we want to specify required and optional Aggregations? My inclination would be to make only a full-featured Distribution required, as it subsumes count, mean, range, and histogram, and sum unless full precision is required.

Reply:
I think it makes sense to have some of them required and some optional. We can add that later I think.